From 42331821ff0e76a125409c3fe3402fceab7fda87 Mon Sep 17 00:00:00 2001
From: Douloureux <Douloureux@users.noreply.github.com>
Date: Wed, 28 Sep 2022 00:00:00 +0000
Subject: [PATCH] disable Omnibox search box stats

Disables sending excess query information in the ?aqs= parameter when performing a search using Google.
---
 components/omnibox/common/omnibox_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/omnibox/common/omnibox_features.cc b/components/omnibox/common/omnibox_features.cc
index 06d7e9c1218e8..0fa2370c0198f 100644
--- a/components/omnibox/common/omnibox_features.cc
+++ b/components/omnibox/common/omnibox_features.cc
@@ -472,7 +472,7 @@ const char kDefaultTypedNavigationsToHttpsTimeoutParam[] = "timeout";
 // Search Results Page URL.
 BASE_FEATURE(kReportAssistedQueryStats,
              "OmniboxReportAssistedQueryStats",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 // If enabled, Omnibox reports the Searchbox Stats in the gs_lcrp= param in the
 // Search Results Page URL.
-- 
2.39.1.windows.1

